### DISCUSSION TITLE
Is the project still worked on?

### DISCUSSION THREAD

**Discussion by NomadicDeveloper22** [dlsc-software-consulting-gmbh/GMapsFX#203] [Q&A] [answered]
Last release doesn't even seem like an update at all

And urls seem to be broken

**Comment by dlemmermann** [comment, marked as answer]
Not actively, but PRs are welcome.

Hunter Austin ***@***.***> schrieb am Sa. 7. Mai 2022 um
06:18:

> Last release doesn't even seem like an update at all
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dlsc-software-consulting-gmbh/GMapsFX/discussions/203>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACIXWXNGYTA73KLYAIAYJNLVIXVIJANCNFSM5VJ2HJ2A>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> com>
>
-- 
Dirk Lemmermann

Software & Consulting
Zurich, Switzerland
+41-(0)79-800-23-20
http://www.dlsc.com
***@***.***
